buildenv/docker/mkdocker.sh

```diff
@@ -621,7 +621,11 @@ install_bootjdks() {
     if [ $version = next ] ; then
       wanted[17]=yes
     else
-      wanted[$version]=yes
+      if [ $version = 18 ] ; then
+        wanted[17]=yes
+      else
+        wanted[$version]=yes
+      fi
     fi
```

Comment:
Please use `elif` instead of the additional nesting:
```
    if [ $version = next ] ; then
      wanted[17]=yes
    elif [ $version = 18 ] ; then
      wanted[17]=yes
    else
      wanted[$version]=yes
    fi
```